roseWong/assignments/slidingwindow/lc209/lc209.py

```diff
@@ -1,11 +1,32 @@
 # Problem Statement #
 
-# Given an array of positive numbers and a positive number ‘S,’ find the length of the smallest contiguous subarray whose sum is greater than or equal to ‘S’. Return 0 if no such subarray exists.
+# Given an array of positive numbers and a positive number ‘S’,
+# find the length of the smallest contiguous subarray whose sum is greater than or equal to ‘S’.
+# Return 0 if no such subarray exists.
+
+# starting with the code from 
```

Comment:
Make sure to write out the high-level psuedo-code of your approach! That way the interviewer can help you with the approach.